setup.cfg

```diff
@@ -79,9 +79,10 @@ openstack.metric.v1 =
 [extras]
 test =
   coverage>=3.6
```

Comment:
Sounds like we can remove that if we depend on pytest-cov

Reply:
done